crates/ide_assists/src/handlers/add_turbo_fish.rs

```diff
@@ -77,13 +78,22 @@ pub(crate) fn add_turbo_fish(acc: &mut Assists, ctx: &AssistContext) -> Option<(
         }
     }
 
+    let number_of_arguments = generics.len();
+    let fish_head = std::iter::repeat("_").take(number_of_arguments).collect_vec().join(",");
```

Comment:
Thanks for very quick implementation!
Just a nitpick: you can use `itertools::repeat_n('_', number_of_arguments).join(",")` for a bit concise code.

Reply:
But wait, I found a Japanese [article](https://maguro.dev/rust-itertools-join/) that collecting into vec firsthand is faster.  So please just forget, sorry.